myuw/templates/handlebars/card/profile/profile_card.html

```diff
@@ -1,145 +1,9 @@
 {% load templatetag_handlebars %}
-{% tplhandlebars "profile_card" %}
-<div class="card" style="position:relative;">
+{% tplhandlebars "common_profile_card" %}
+<div class="card common-profile" style="position:relative;">
+  <h2>{{#if has_preferred}}{{display_name}} <span>({{full_name}})</span>{{else}}{{full_name}}{{/if}}  <span><a href="https://identity.uw.edu/"><i class="fa fa-pencil"></i></a></span></h2>
```

Comment:
@mikeseibel Can we add a `title="Manage your preferred name"` to the <a> tag (pencil icon) here?